misaligned.c

```diff
@@ -1,21 +1,28 @@
 #include <stdio.h>
-#include <assert.h>
+#include <string.h>
+#include "misaligned.h"
+
+ColorPair colorPairList[MAX_COLOR_PAIR_COUNT];
+const char* majorColor[] = {"White", "Red", "Black", "Yellow", "Violet"};
+const char* minorColor[] = {"Blue", "Orange", "Green", "Brown", "Slate"};
+
+void printColorMap() {
+    int size, i;
+    size = fillColorMapList();
+
+    for (i = 0; i < size; i++) {
+        printf("%d | %s | %s\n", i, colorPairList[i].majorColor, colorPairList[i].minorColor);

```

Comment:
zero based color pair is printed... is this expected from a user point of view? or the color pair number needs to start from 1?
Also if that is the case, is there a test case to confirm that expectation?